shared/surface_flux.F90

```diff
@@ -173,9 +173,9 @@ module surface_flux_mod
                                            !! heat.  This option is available for legacy purposes, and is not recommended for
                                            !! new experiments.
 logical :: ncar_ocean_flux_multilevel  = .false. !< Use NCAR climate model turbulent flux calculation described by Large and Yeager, allows for different reference height for wind, temp and spec. hum.
-real :: bulk_zu                           !< Reference height for wind speed
-real :: bulk_zt                           !< Reference height for atm temperature
-real :: bulk_zq                           !< Reference height for atm humidity
+real :: bulk_zu = 10.                      !< Reference height for wind speed (meters)
```

Comment:
How did you determine that the default value should be 10?

Reply:
In reanalysis datasets, such as JRA, the winds, temp, etc are given at 10 meters above the sea surface. Whoever is running models with ncar_ocean_flux=.true. , has to override these defaults in the namelist.